File: HelperJobby/DTOs/UserJobInteractions/InterviewDTO.cs
using HelperJobby.DTOs.Account;
using HelperJobby.DTOs.Job;

namespace HelperJobby.DTOs.UserJobInteractions;

public class InterviewDTO
{
    public int JobId { get; set; }

    public int JobSeekerId { get; set; }

    public JobDTO Job { get; set; }

    public JobSeekerDTO JobSeeker { get; set; }

    public DateTime InterviewStart { get; set; }
    public TimeOnly InterviewEnd { get; set; }
    public string InterviewType { get; set; }
    public string AppointmentInfo { get; set; }
}

File: HelperJobby/client-app/src/DTOs/userJobInteractionsDTOs/InterviewDTO.ts
import {JobSeekerDTO} from "../accountDTOs/JobSeekerDTO";
import {JobDTO} from "../jobRelatetedDTOs/JobDTO";

export interface InterviewDTO {
    jobId: number;
    jobSeekerId: number;
    job: JobDTO;
    jobSeeker: JobSeekerDTO;
    interviewStart: string;
    interviewEnd: string;
    interviewType: string;
    appointmentInfo: string;
}
